packages/runtime/container-runtime/src/summaryManager.ts

```diff
@@ -259,7 +195,7 @@ export class SummaryManager extends EventEmitter implements IDisposable {
             return { shouldSummarize: false, stopReason: "parentShouldNotSummarize" };
         } else if (this.disposed) {
             return { shouldSummarize: false, stopReason: "disposed" };
-        } else if (this.quorumHeap.getSummarizerCount() > 0) {
+        } else if (this.orderedClients.getSummarizerCount() > 0) {
```

Comment:
what if the zombie client is a summarizer client. i think orderedClients needs to track summarizers, and allow incrementing past them, which should also drop the count

Reply:
Yeah this logic relying on summarizer count needs to be reworked/removed when we actually start shifting the elected client. That is mentioned in the issue here: https://github.com/microsoft/FluidFramework/issues/5222#issue-813818343 as the first bullet point. I left it as-is since right now we are just doing logging. I don't see any reason to have them in the linked list though.

Reply:
can you add a comment here